packages/node/src/configure/configure.module.ts

```diff
@@ -42,7 +44,33 @@ function defaultSubqueryName(config: Partial<IConfig>): MinConfig {
   } as MinConfig;
 }
 
-const logger = getLogger('configure');
+// Check if a subquery name is a valid schema name
+function validDbSchemaName(name: string): boolean {
+  if (name.length === 0) {
+    return false;
+  } else {
+    const regexp = new RegExp('[a-zA-Z_]');
```

Comment:
Can we make it more strict than the actual postgres's rule. like `/^[a-zA-Z_][a-zA-Z0-9_\-\/]{0,62}$/`

Reply:
Used your regex as I'm happy with it